test_e2e.sh

```diff
@@ -1072,6 +1074,81 @@ function e2e::exechook_fail_retry() {
     fi
 }
 
+##############################################
+# Test exechook-success with GIT_SYNC_ONE_TIME
```

Comment:
comment should go on the real test fn, the helper can just describe args and purpose

Reply:
N/A anymore since removing helper function since only running test once.